packages/cozy-client/examples/helpers/file.js

```diff
@@ -0,0 +1,116 @@
+const { default: CozyClient, QueryDefinition } = require('cozy-client')
+const fs = require('fs')
+const { FileDocument } = require('cozy-client/dist/types')
+
+const schema = {
+  files: {
+    doctype: 'io.cozy.files'
+  }
+}
+
+const generateFakeFolder = () => ({
+  name: 'folder-name' + Math.random(),
+  icon: 'icon',
```

Comment:
I don't think `icon` is an expected directory field: https://github.com/cozy/cozy-stack/blob/master/docs/files.md#post-filesdir-id